raghu_bdc.md

```diff
@@ -0,0 +1,10 @@
+ # New Albany, OH
+
+ ### Restaurants
+Cake Walk
+Dosa Corner
```

Comment:
What kind of food do these restaurants serve? What area are they in?

Reply:
Freshly prepared :cake: indo-Chinese food and :bread: